### DISCUSSION TITLE
hyper::Body::channel in hyper 1

### DISCUSSION THREAD

**Discussion by DreamTexX** [hyperium/hyper#3159] [General]
Is there some equivalent to `hyper::Body::channel` in Version 1?

**Comment by seanmonstar** [comment]
There is not, _yet_. This issue describes the desire to add one to `hyper-util`: https://github.com/hyperium/hyper/issues/2858 (all are welcome to help :)).

**Reply by DreamTexX** [reply]
Ok, thanks! Guess I will stay on 0.14 for now then. Or is there a way to wrap an async pipe writer into a body? 😅

**Reply by seanmonstar** [reply]
I'm not sure what that is, but there is `http_body_util::StreamBody`, which might what you're looking for.
